python/paddle/fluid/contrib/tests/test_calibration.py

```diff
@@ -19,10 +19,8 @@
 import random
 import paddle
 import paddle.fluid as fluid
-import argparse
 import functools
 import contextlib
-import paddle.fluid.profiler as profiler
 from paddle.dataset.common import download
 from PIL import Image, ImageEnhance
```

Comment:
please update line 44 '# TODO(guomingz): Remove duplicated code from line 45 ~ line 114'->
'# TODO(guomingz): Remove duplicated code from resize_short, crop_image, process_image, _reader_creator'.
The reason is that the line number changes in this PR.